common/experiment_utils.py

```diff
@@ -19,7 +19,7 @@
 from common import environment
 
 # Time interval for collecting experiment data (e.g. corpus, crashes).
-SNAPSHOT_PERIOD = 15 * 60  # Seconds.
+SNAPSHOT_PERIOD = environment.get('SNAPSHOT_PERIOD', 15 * 60)  # Seconds.
```

Comment:
Can we keep this a constant, and move the caller to use environment.get. Not good to set dynamic vars through global.

Reply:
I'll probably replace with a function `get_snapshot_seconds()`

Reply:
Did this.